vllm/compilation/backends.py

```diff
@@ -55,9 +175,84 @@ def wrap_inductor(graph,
     # inductor can inplace modify the graph, so we need to copy it
     # see https://github.com/pytorch/pytorch/issues/138980
     graph = copy.deepcopy(graph)
-    compiled_graph = compile_fx(graph,
-                                example_inputs,
-                                config_patches=current_config)
+
+    cache_data = compilation_config.inductor_hash_cache
+    if (runtime_shape, graph_index) in cache_data:
+        # we compiled this graph before
+        # so we can directly lookup the compiled graph via hash
+        hash_str = cache_data[(runtime_shape, graph_index)]
+        logger.debug(
+            "directly lookup the %s-th graph for shape %s via hash %s",
+            graph_index, str(runtime_shape), hash_str)
```

Comment:
Could you add some info-level logging as well so we always know whether caching is/isn't being used from looking at the logs?

Can use `print_info_once` so it doesn't print for every layer
```
print_info_once("Using previously compiled compute graph for shape %s", hash_str)
```

Reply:
added in [c4478c8](https://github.com/vllm-project/vllm/pull/11108/commits/c4478c8a4c67251fb6475986bd6bf208371aae7a)